tests/py/test_security.py

```diff
@@ -42,3 +47,25 @@ def test_ahtr_sets_x_content_type_options(self):
     def test_ahtr_sets_x_xss_protection(self):
         headers = self.client.GET('/about/').headers
         assert headers['X-XSS-Protection'] == '1; mode=block'
+
+
+    # ep - EncryptingPacker
+
+    packed = b'gAAAAABXJMbdriJ984uMCMKfQ5p2UUNHB1vG43K_uJyzUffbu2Uwy0d71kAnqOKJ7Ww_FEQz9Dliw87UpM'\
+             b'5TdyoJsll5nMAicg=='
+
+    def test_ep_packs_encryptingly(self):
+        packed = Participant.encrypting_packer.pack({"foo": "bar"})
+        assert urlsafe_b64decode(packed)[0] == b'\x80'  # Fernet version
+
+    def test_ep_unpacks_decryptingly(self):
+        assert Participant.encrypting_packer.unpack(self.packed) == {"foo": "bar"}
```

Comment:
Might be worth checking whether `encrypting_packer` decrypts data encrypted with old keys too?